src/power_grid_model/core/options.py

```diff
@@ -17,16 +17,21 @@ class OptionSetter:
     setter for options
     """
 
+    _value: Any
     _setter: Callable
 
     def __init__(self, setter):
         self._setter = setter
 
     def __set__(self, instance: "Options", value: Any):
         self._setter(instance.opt, value)
+        self._value = value
 
-    def __get__(self, instance, owner):
-        raise NotImplementedError("Cannot get option value!")
+    def __get__(self, instance, owner) -> Any:
```

Comment:
You are adding this back again in this commit...

Reply:
huh, weird :\ thanks for catching this